docs/versioned_docs/version-zenith/user/quickstart/snippets/hello/main.go

```diff
@@ -0,0 +1,16 @@
+package main
+
+import (
+	"context"
+)
+
+type Mymodule struct{}
+
+// say hello
+func (m *Mymodule) Hello(ctx context.Context) string {
+	version, err := dag.Container().From("node:18-slim").WithExec([]string{"node", "-v"}).Stdout(ctx)
+	if err != nil {
+		panic(err)
+	}
+	return ("Hello from Dagger and Node " + version)
+}
```

Comment:
```suggestion
func (m *Mymodule) Hello(ctx context.Context) (string, error) {
	version, err := dag.Container().From("node:18-slim").WithExec([]string{"node", "-v"}).Stdout(ctx)
	if err != nil {
		return "", err
	}
	return ("Hello from Dagger and Node " + version), nil
}
```

While the `panic` technically works, returning errors explicitly is more "standard go"

Reply:
+1